docs/from_sql_setup.md

```diff
@@ -0,0 +1,221 @@
+# Quick Steps to Setup SQL Databases for Flux Testing
+
+## Timescale
```

Comment:
We need to rework or remove the README from `master` for the SQL setup

Reply:
I agree it doesn't really have a "place"  but it is a very nice bit of text that should be made available to the community somehow...maybe a blog?